api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/convolutional/AbstractConv.kt

```diff
@@ -19,6 +19,35 @@ import org.tensorflow.op.core.Variable
 import java.lang.IllegalArgumentException
 import kotlin.math.roundToInt
 
+/**
+ * Abstract Convolutional layer is a base block for building base types of convolutional layers
+ * of any dimensionality. It should simplify the internal calculations needed in most of
+ * the convolutional layers and abstract the process of naming weights for these layers. It keeps
+ * the actual implementation of convolutional layers e.i. the kernel and bias learnable variables
```

Comment:
minor typo in line 26: "e.i." should be " i.e."

Reply:
thanks for pointing this, just fixed. What about the essence of this doc - was it readable for you @dosier?